packages/terra-abstract-modal/package.json

```diff
@@ -32,6 +32,7 @@
     "keycode-js": "^3.1.0",
     "mutationobserver-shim": "<=0.3.3",
     "prop-types": "^15.5.8",
+    "react-focus-on": "^3.9.1",
```

Comment:
Are we now introducing a new third-party library?  If so, how was this vetted?  Why do the other two lock libraries not work?

Reply:
Hi @cm9361 
We found issues with react-focusTrap in certain scenarios, like form-single select inside the modal popup, As a solution, we attempted to use react-focusLock. However, we found that the closeOnOutsideClick functionality wasn't functioning properly. We then explored **focusOn**, which provided support for all the required functionalities.

Reply:
Do we need to also expose this closeOnOutsideClick prop in ModalManager.  I don't think users of that component, such as MPages Fusion, can set that value as-is.